server/src/inmantals/server.py

```diff
@@ -89,8 +94,22 @@ def sync_compile_and_anchor() -> None:
             resources.resource.reset()
             handler.Commander.reset()
 
+            project_signature = inspect.signature(Project.__init__)
             # fresh project
-            Project.set(Project(self.rootPath))
+            if (
+                "venv_path" not in project_signature.parameters.keys()
+                and self.compiler_venv_path
+            ):
+                logger.info(
+                    "Compiler virtual env path specified, but "
+                    "the current version of Inmanta doesn't support it, using the default"
+                )
```

Comment:
This line will produce an incorrect log message. The `extension.ts` makes sure that the value of `compiler_venv_path` is set an a non-empty string. So this log line will report that the report that `compiler_venv_path` is set while it isn't.